pages/price-feeds/how-pyth-works/why-update-prices.mdx

```diff
@@ -0,0 +1,17 @@
+# Why Update Prices
+
+Pyth uses a pull-based oracle model. Unlike traditional push oracles that automatically update prices on-chain at regular intervals, Pyth requires users to explicitly update the on-chain price before reading it.
```

Comment:
uses a pull oracle model -- we don't say "pull-based"

Reply:
pls learn this for future references